c3/lib_superlearn/superlearn_util.c

```diff
@@ -261,6 +261,41 @@ sl_mem_manager_alloc(size_t d, size_t n,
     
 }
 
+/***********************************************************//**
```

Comment:
All these functions should have checks to make sure mem is not NULL, if it is NULL you can return NULL